code/game/machinery/computer/computer.dm

```diff
@@ -19,6 +19,12 @@
 	overlay_layer = layer
 	..()
 
+/obj/machinery/computer/Destroy()
```

Comment:
@davipatury 

This caused some runtimes with computers, and I"ll tell you why.

Basically, our construction is handled quite a bit differently than TG, and the `circuit` path is nothing more than a placeholder; the actual circuit isn't stored in the computer.

This is something you may also have to refactor before doing the full object damage system port, as this will probably cause problems, later on, down the line.